### DISCUSSION TITLE
Static Missing on Dia/Baal

### DISCUSSION THREAD

**Discussion by tyson-dev** [isid0re/SoloLeveling#335] [General]
Next Thing i noticed i see my Sorc correctly switching and choosing static (skill at bottom right visible) but dia and baal don't lose 25% in norm around level 32/33 ( while leveling to meet nightmare specs). Maybe she's not using static at all, idk yet.

Can anyone check your cold-sorc ( it's cold sorc here, probably other sorcs too) and confirm/negate please.

Edit: same on nightmare Andy, doesn't lose health from static.

Another Edit: It looks fine now some time into Nightmare, maybe she did respec i dunno :D

**Comment by isid0re** [comment]
Tyson,

Thanks for posting. I'm actually rewriting all of the class attackoverrides for all the classes. So I will take a look into this as well.

**Comment by tyson-dev** [comment]
Hi Isid0re, yeah looks like this is a thing before final respec, Assa is also not boing ( she keeps switching to bo every three teleports @ level 66 without actually casting it), cold sorc also not dealing with immunes around level 70-80
